common/app/model/Formats.scala

```diff
@@ -4,7 +4,7 @@ import common.{NavItem, SectionLink, Pagination}
 import model.content._
 import model.facia.PressedCollection
 import model.liveblog.{BlockAttributes, BodyBlock}
-import quiz._
+import quiz.{Image => _, _}
```

Comment:
The first import excludes `Image`, the second import imports any remaining symbols. So that reads  import everything but `Image`.